src/gui/userstatusselectormodel.cpp

```diff
@@ -141,23 +141,23 @@ void UserStatusSelectorModel::onError(UserStatusConnector::Error error)
         return;
 
     case UserStatusConnector::Error::CouldNotFetchUserStatus:
-        setError(tr("Could not fetch user status. Make sure you are connected to the server."));
+        setError(tr("Could not fetch status. Make sure you are connected to the server."));
         return;
 
     case UserStatusConnector::Error::UserStatusNotSupported:
-        setError(tr("User status feature is not supported. You will not be able to set your user status."));
+        setError(tr("Status feature is not supported. You will not be able to set your status."));
         return;
 
     case UserStatusConnector::Error::EmojisNotSupported:
-        setError(tr("Emojis feature is not supported. Some user status functionality may not work."));
+        setError(tr("Emojis feature is not supported. Some status functionality may not work."));
```

Comment:
We could make it slightly shorter, maybe? "Emojis are not supported. Some status functionality may not work."

Reply:
Yeah, less verbose that way -- changed it